### PR TITLE
{Duplicate of #5664] Understanding how cluster markers work

### LINKED ISSUE #4841: Understanding how cluster markers work in web mapping using leaflet.js
#### Proposed title of article
[Web-GIS] Understanding how cluster markers work in web mapping using leaflet.js

#### Proposed article introduction
Understanding how cluster markers work in leaflet.js

As we all know Big Data is the keyword for the 21st century. A few people realize that a substantial part of these massive quantities of data is categorized as geospatial. Nowadays almost all the devices we used are satellite-based radio navigation systems like GPS.

Nonetheless even today, the simplest case of displaying hundreds of Points on a web map with acceptable usability can be a real challenge. Especially from the perspective of the user, the navigation experience can be utterly painful. For instance, [smartrak](http://www.smartrak.co.nz/), this platform has customers with thousands of points on the map. When you zoom it out, these markers all overlap and make the map looks messy and crowded.

To improve this, it is professional to use marker clustering, a technique of grouping markers that are close to each other together on each zoom level.

#### Key takeaways 
-Getting started with leaflet.js map
-Animated map interaction
-Group markers into clusters

#### Article quality
This article will help understand how to group markers close to each other and improve the look and interaction of websites like [smartrak](http://www.smartrak.co.nz/), 

#### References
[Leaflet.js](https://leafletjs.com)
[Marker Cluster](https://leafletjs.com/2012/08/20/guest-post-markerclusterer-0-1-released.html)

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

### PR DESCRIPTION
**Pull request by Adebogunabdulroheem**
This closes #4841

### CODE REVIEW

**Comment by Adebogunabdulroheem** [comment]
@jumamiller I have made changes to this PR . Please help check it

**Comment by ahmadmardeni1** [comment]
If this article still needs any changes, please close it. @jumamiller

File: content/articles/understanding-how-cluster-marker-works/index.md [hunk 1]
```diff
@@ -0,0 +1,232 @@
+### Understanding how cluster markers work in web mapping using leaflet.js
+  This isn't breaking news; we all know that big data is the buzzword of the twenty-first century, and it's surprising to learn that only a few individuals recognize that a major portion of the data is classified as geographical. Almost all of the devices we use nowadays are satellite-based systems like GPS. Today, even the most basic scenario of showing data and markers on a map with a satisfactory user experience can be a major issue, especially for the user, who may find it difficult to grasp and analyze data at a certain zoom level. For example, on a platform with hundreds of users, when you zoom out the map, these markings all overlap, making the map appear cluttered and crowded. The marker clustering method is recommended for improving this type of map.
```
Comment (jumamiller) [inline content/articles/understanding-how-cluster-marker-works/index.md:2]: This paragraph is extremely long.

File: content/articles/understanding-how-cluster-marker-works/index.md [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+### Objective
+We'll create a map with clustering features at the end of this guide, which will include: - Leaflet map and its usage.
+- A lovely animation of a Leaflet Marker Cluster.
+
+### How to Get Started
+Have you ever attempted to use a map to visualize a big quantity of data? If you use the traditional way, you'll wind up with something like this, with a lot of markers strewn about.
+
+(engineering-education/understanding-how-cluster-marker-works/marker.png)![image info](engineering-education/understanding-how-cluster-marker-works/marker.png)
+
+Instead of utilizing the traditional approach to visualize a map, consider employing the clustering method, which is the most efficient way of mapping when it comes to viewing large amounts of data. The leaflet-marker-cluster plugin is required to achieve marker clustering. So let's get started because we need to set up our data visualization map. Create an 'index.html' file in a new project directory with any code editor, then copy and paste the snippets below.
```
Comment (jumamiller) [inline content/articles/understanding-how-cluster-marker-works/index.md:17]: This one too, kindly go thro' the entire article and ensure your article flows well.Avoid wordy/ long paragraphs as much as possible!

File: content/articles/understanding-how-cluster-marker-works/index.md [hunk 3]
```diff
@@ -0,0 +1,211 @@
+### Understanding how cluster markers work in web mapping using leaflet.js
+ Big data is the buzzword of the twenty-first century. Only a few people recognize that a major portion of the data is classified as geographical. Even the most basic scenario of showing data and markers on a map with a satisfactory user experience can be a major issue. The marker clustering method is recommended for improving this type of map.
```
Comment (hectorkambow) [inline content/articles/understanding-how-cluster-marker-works/index.md:2]: A statement like this needs a source... and I'm more than half way into your introduction and have no clue what the article is about. @Adebogunabdulroheem

**Comment by hectorkambow** [comment]
Closing this PR as it seems a bit of a ways away from being publish ready.

**Pull request closed without merging**